client/my-sites/backup/rewind-flow/granular-restore.tsx

```diff
@@ -56,6 +61,76 @@ const BackupGranularRestoreFlow: FunctionComponent< Props > = ( {
 		getInProgressBackupForSite( state, siteId )
 	);
 
+	const [ showFiles, setShowFiles ] = useState< boolean >( true );
+	const [ showAllFiles, setShowAllFiles ] = useState< boolean >( false );
+	const [ showTables, setShowTables ] = useState< boolean >( true );
+	const [ showAllTables, setShowAllTables ] = useState< boolean >( false );
+	const [ showThemes, setShowThemes ] = useState< boolean >( true );
+	const [ showAllThemes, setShowAllThemes ] = useState< boolean >( false );
+	const [ showPlugins, setShowPlugins ] = useState< boolean >( true );
+	const [ showAllPlugins, setShowAllPlugins ] = useState< boolean >( false );
```

Comment:
Given that TypeScript can infer the type of the state from the initial value, the explicit type annotation `< boolean >` is redundant. We could simplify these lines to:

```suggestion
	const [ showFiles, setShowFiles ] = useState( true );
	const [ showAllFiles, setShowAllFiles ] = useState( false );
	const [ showTables, setShowTables ] = useState( true );
	const [ showAllTables, setShowAllTables ] = useState( false );
	const [ showThemes, setShowThemes ] = useState( true );
	const [ showAllThemes, setShowAllThemes ] = useState( false );
	const [ showPlugins, setShowPlugins ] = useState( true );
	const [ showAllPlugins, setShowAllPlugins ] = useState( false );
```

By doing this, the code becomes a bit cleaner without losing any of the type safety that TypeScript provides. What do you think?

Reply:
Sounds good 👍